guide/onboarding/getting-to-know-your-users.md

```diff
@@ -0,0 +1,95 @@
+* How did they hear about my application? Word of mouth? App store search?
+* Do first-time users have specific needs or goals?
+
+Perhaps after research, you identify your users as "cypherpunks" with sound knowledge of Bitcoin, but they are new to your app. You might build an experience that preserves as much of their privacy as possible and is super customizable from the outset.
+
+Aim to gather as much information and observations as possible in order to empathize with your users.
+
+## Identify barriers and motivators they have
+You can then use this research to identify where potential problems may exist. This will then allow you to highlight opportunities for innovation.
+
+### Rookie example
+
+Let's use an example of saving for the future and see how we can understand a user with this goal. For the sake of simplicity let's focus on one persona or maybe more accurately personal story.
+
+> Joshua Akinsola is a 21-year old Nigerian studying economics in the large city of Lagos. Growing up, his family struggled to save any money, due to a lack of financial education, high unemployment and a local currency that is inflating at 15% per year. There is a lack of reliable savings technology in his country so he opts to save money for the future using bitcoin.
+
+The uncovering of this scenario could be the result of a user interview[^3], that gathers insights into their needs and frustrations. By focusing on the emotional aspect of saving using bitcoin we may identify the following themes for barriers and motivations.
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/barriers-motivators.svg"
+   alt-text = "Barriers & motivators"
+   width = 1600
+   height = 800
+%}
+
+#### Barriers (strongest to weakest)
+
+* Fear of losing private key
+    * "I'm scared that I will lose my private key and lose my savings forever"
+* Fear of being hacked
+    * "How will I protect myself if I get targetted by a hacker?"
+* Fear of being judged
+    * "Bitcoin is still new, will people think I'm weird for saving like this?"
+
+#### Motivators (strongest to weakest)
+* Security
+    * "I want to have some security about my future, so I can own a home and send my kids to school"
+* Control
+    * "I want to be in total control of my money, I don't trust my government to keep inflation low"
+* Knowledge
+    * "I think it's the future so it makes sense to learn more about it"
+
+You might also uncover barriers and motivators in various channels such as chats, issues or even twitter and find insights this way.
+
+## Brainstorm solutions with your team
+With this information, you can weaken the barriers and strengthen the motivators to craft the best experience for a user like Josh.
+
+To weaken Josh's "Fear of losing a private key," a wallet might opt to initially backup an encrypted version of his recovery phrase to a cloud provider and educate him over time on how to take more control of his private key.
```

Comment:
I think you can also point out the savings use-case here and how that puts a bigger focus on security.

Reply:
Actually, I think there might be more information you can extract from this persona, to give the reader an example of how complex this can get.
- What tech does a 21-year old in Nigeria have access to?
- What type of wallet are they likely to use based on their language, location, etc?
- How big is the amount he's likely to save?
- What are unique security concerns in this culture/context?
- What about localization?
- How do Nigerians typically convert their fiat to Bitcoin?

You might not have to list it all out, but by showing a range of information that can be used, readers gets a sense of how deep they can go.